docs/Installation.md

```diff
@@ -0,0 +1,93 @@
+# Installation Guide for ClusterLink
+The ClusterLink gateway contains three main components: the control-plane, data-plane, and gwctl (more details can be found [here](../README.md#what-is-clusterlink)).
+
+ClusterLink can be deployed in a local environment (Kind) or any remote K8s cluster environment (GKE, EKS, IBM-K8S, etc.).
+
+To deploy the ClusterLink gateway in a K8s cluster, follow these steps:
+
+1. Create and deploy certificates and the ClusterLink Deployment file.
+2. Expose the ClusterLink deployment to a public IP.
+3. Optionally, create a central gwctl component to control all gateways.
+
+Before you begin, please clone and build the project according to the [instructions](../README.md#building-clustelink).
+### 1. ClusterLink Deployment and certificates.
+In this step, we build the ClusterLink certificates and deployment files.
+
+1) Create a folder (eg. TEST_DIR) for all deployments and yaml files:
+
+        export PROJECT_DIR=`git rev-parse --show-toplevel`
+        export TEST_DIR=$PROJECT_DIR/bin/test/
+        mkdir -p $TEST_DIR
+        cd $TEST_DIR
+
+2) Create Fabric certificates, including the Root CA certificate for all the clusters (This step needs to be done only once per fabric).:
+
+        $PROJECT_DIR/bin/cl-adm create fabric
+
+3) Create certificates and a deployment file for the cluster (e.g., peer1). This step needs to be performed for every K8s cluster.:
+
+        $PROJECT_DIR/bin/cl-adm create peer --name peer1
+
+    Note: By default, the images are pulled from ghcr.io/clusterlink-net. To change the container registry, you can use the ```--container-registry``` flag.
+
+4) Apply ClusterLink deployment:
+   
+        kubectl apply -f $TEST_DIR/peer1/k8s.yaml
+
+
+5) Verify that all components (cl-controlplane, cl-dataplane, gwctl) are set up and ready.
+   
+        kubectl get pods
+
+### 2. Expose the ClusterLink deployment to a public IP
+Create public IP for accessing the ClusterLink gateway.
+For Local environment create a K8s nodeport service:
+
+        kubectl apply -f $PROJECT_DIR/demos/utils/manifests/kind/cl-svc.yaml
```

Comment:
```suggestion
        kubectl expose service cl-dataplane --type=NodePort --name=cl-dataplane-nodeport
```
Also, missing some `export PEER1_PORT=`kubectl get service cl-dataplane-nodeport -o "jsonpath=?"`

Reply:
Did you try this? it is not working...  the nodeport  field is missing

Reply:
I left it for your to figure out :)
But anyhow, this works:
```
export PEER1_PORT=`kubectl get service cl-dataplane-nodeport -o "jsonpath={.spec.ports[0].nodePort}"``
```